dev/config/keycloak/import/ltt-realm.json

```diff
@@ -0,0 +1,1463 @@
+{
```

Comment:
What might I compare this with to identify what's specific to Let's Talk Tech?

Reply:
Not 100% clear if you're referring to the project name, or the cPRO config :sweat_smile: 

I copied/pasted the default `master` realm to create the `ltt` realm in this PR (65912bc), but you'll want to look at #3 for all the LTT/cPRO-specific changes

Reply:
Thanks, that realm history is what I was interested in. I wasn't referring to dhair2, but the larger project.